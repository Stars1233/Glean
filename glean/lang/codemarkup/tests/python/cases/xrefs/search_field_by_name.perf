{
  "@generated": null,
  "facts_searched": { "python.SearchFieldByName.4": 6 },
  "full_scans": [ "python.SearchFieldByName.4" ]
}