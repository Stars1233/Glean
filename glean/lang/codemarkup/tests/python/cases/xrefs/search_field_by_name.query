query: python.SearchFieldByName { name = _ }
perf: true
transform: [gensort, []]
