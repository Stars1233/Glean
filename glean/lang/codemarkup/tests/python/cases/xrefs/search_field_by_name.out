[
  "@generated",
  {
    "key": {
      "name": "class_var",
      "parent": {
        "key": {
          "local_name": { "key": "HelperClass" },
          "parent": { "key": { "local_name": { "key": "lib" } } }
        }
      },
      "decl": { "key": { "name": { "key": "lib.HelperClass.class_var" } } }
    }
  },
  {
    "key": {
      "name": "cls",
      "parent": {
        "key": {
          "local_name": { "key": "<locals>" },
          "parent": {
            "key": {
              "local_name": { "key": "class_method" },
              "parent": {
                "key": {
                  "local_name": { "key": "HelperClass" },
                  "parent": { "key": { "local_name": { "key": "lib" } } }
                }
              }
            }
          }
        }
      },
      "decl": {
        "key": {
          "name": { "key": "lib.HelperClass.class_method.<locals>.cls" }
        }
      }
    }
  },
  {
    "key": {
      "name": "instance_var",
      "parent": {
        "key": {
          "local_name": { "key": "HelperClass" },
          "parent": { "key": { "local_name": { "key": "lib" } } }
        }
      },
      "decl": { "key": { "name": { "key": "lib.HelperClass.instance_var" } } }
    }
  },
  {
    "key": {
      "name": "old_style_class_var",
      "parent": {
        "key": {
          "local_name": { "key": "HelperClass" },
          "parent": { "key": { "local_name": { "key": "lib" } } }
        }
      },
      "decl": {
        "key": { "name": { "key": "lib.HelperClass.old_style_class_var" } }
      }
    }
  },
  {
    "key": {
      "name": "self",
      "parent": {
        "key": {
          "local_name": { "key": "<locals>" },
          "parent": {
            "key": {
              "local_name": { "key": "abstract_method" },
              "parent": {
                "key": {
                  "local_name": { "key": "AbstractClass" },
                  "parent": { "key": { "local_name": { "key": "lib" } } }
                }
              }
            }
          }
        }
      },
      "decl": {
        "key": {
          "name": { "key": "lib.AbstractClass.abstract_method.<locals>.self" }
        }
      }
    }
  },
  {
    "key": {
      "name": "self",
      "parent": {
        "key": {
          "local_name": { "key": "<locals>" },
          "parent": {
            "key": {
              "local_name": { "key": "method" },
              "parent": {
                "key": {
                  "local_name": { "key": "HelperClass" },
                  "parent": { "key": { "local_name": { "key": "lib" } } }
                }
              }
            }
          }
        }
      },
      "decl": {
        "key": { "name": { "key": "lib.HelperClass.method.<locals>.self" } }
      }
    }
  }
]